meta-balena-common/recipes-bsp/u-boot/u-boot_%.bbappend

```diff
@@ -3,8 +3,8 @@ FILESEXTRAPATHS_append := ":${THISDIR}/files"
 # We require these uboot config options to be enabled for env_resin.h
 SRC_URI += "file://balenaos_uboot.cfg"
 
-SRC_URI += "${@bb.utils.contains('DISTRO_FEATURES', 'development-image', '', 'file://balenaos_uboot_prod.cfg', d)}"
-SRC_URI += "${@bb.utils.contains('OS_DEV_UBOOT_DELAY', '1', 'file://balenaos_uboot_delay.cfg', 'file://balenaos_uboot_nodelay.cfg', d)}"
+SRC_URI += "${@bb.utils.contains('DISTRO_FEATURES', 'osdev-image', '', 'file://balenaos_uboot_prod.cfg', d)}"
+SRC_URI += "${@bb.utils.contains('DISTRO_FEATURES', 'osdev-image', 'file://balenaos_uboot_delay.cfg', 'file://balenaos_uboot_nodelay.cfg', d)}"
```

Comment:
Just wanted to check whether the second entry should have 'OS_DEV_UBOOT_DELAY' and '1' somewhere like the original?

Reply:
Before we were using the `OS_DEV_UBOOT_DELAY` variable to configure the uboot delay, and the `development-image` feature to configure the silent u-boot mode or not.
With the changes we use use `osdev-image` for both as it's simpler. So you either get a production image which is silent and has no u-boot delay or an OS development one that is non-silent and waits in u-boot.